x-pack/plugins/security_solution/public/users/components/kpi_users/translations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const ENABLE_USER_RISK_TEXT = i18n.translate(
+  'xpack.securitySolution.kpiUser.enableUserRiskText',
+  {
+    defaultMessage: 'Enable user risk module to see more data',
+  }
+);
+
+export const LEARN_MORE = i18n.translate('xpack.securitySolution.kpiUser.learnMore', {
+  defaultMessage: 'Learn more about',
+});
+
+export const USER_RISK_DATA = i18n.translate('xpack.securitySolution.kpiUser.userRiskData', {
+  defaultMessage: 'user risk data',
```

Comment:
is this supposed to be all lowercase?

Reply:
It supposes to be lowercase. It doe look weird because I am waiting for the documentation link. In the end, it should look like this:
```tsx
<FormattedMessage id="id1"
  defaultMessage="Learn more about {userRiskScoreDocumentationLink}"
  values={{
      userRiskScoreDocumentationLink: (
        <EuiLink href={RISKY_USERS_DOC_LINK} target="_blank">
          <FormattedMessage
            id="id2"
            defaultMessage="'user risk data'"   <------------ HERE
          />
        </EuiLink>
      ),
  }}
```